packages/framework-provider-aws/package.json

```diff
@@ -20,8 +20,8 @@
     "url": "git+https://github.com/boostercloud/booster.git"
   },
   "dependencies": {
-    "@boostercloud/framework-common-helpers": "^0.30.4",
-    "@boostercloud/framework-types": "^0.30.4"
+    "@boostercloud/framework-common-helpers": "*",
+    "@boostercloud/framework-types": "*"
```

Comment:
I'm not sure how this could work in the long term 🤔 If every package only requires the presence of the others, but it doesn't care which version, it could happen that npm eventually installs, let's say Booster CLI v 1.20.4, but it's okay with framework types 0.24.0 because there are no version restrictions. If we want to allow different packages to have different versions, I think we should set at least a minimum version here to guarantee that the package is at least using a compatible dependency. Still, I understand why allowing different packages to have different versioning schemas, as we're publishing new versions that are essentially the same package as before when a new version has no changes, but keeping all core packages in sync also helps to guarantee that a specific bundle works well together. What do you think?

Reply:
Answered as a PR comment :)